src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -75,6 +75,8 @@ def load_arguments(self, _):
             c.argument('disable_key_based_metadata_write_access', arg_type=get_three_state_flag(), help="Disable write operations on metadata resources (databases, containers, throughput) via account keys")
             c.argument('enable_public_network', options_list=['--enable-public-network', '-e'], arg_type=get_three_state_flag(), help="Enable or disable public network access to server.")
             c.argument('enable_analytical_storage', arg_type=get_three_state_flag(), help="Flag to enable log storage on the account.", is_preview=True)
+            c.argument('network_acl_bypass', options_list=['--network-acl-bypass'], help="Flag to enable or disable Network Acl Bypass.")
+            c.argument('network_acl_bypass_resource_ids', nargs='+', options_list=['--network-acl-bypass-resource-ids'], help="List of Resource Ids to allow Network Acl Bypass.")
```

Comment:
could we add a shorter option for the parameter in your options list? e.g. `-i`.

Reply:
Done, thanks!